index.html

```diff
@@ -19,12 +19,15 @@
         <a class="logo-link" href="#">
           <h2 class="logo">My Logo</h2>
         </a>
-        <ul class="desk-top-menu">
-          <li><a href="#">Portfolio</a></li>
-          <li><a href="#">About</a></li>
-          <li><a href="#">Contact</a></li>
+        <ul id="mobile_navbar" class="desk-top-menu">
+          <li class="close-menu-btn" id="close-menu-btn">
+            <img src="./design/assets/closeBtn.svg" alt="close button" />
+          </li>
+          <li class="mob-nav-item"><a href="#works-container">Portfolio</a></li>
+          <li class="mob-nav-item"><a href="#about-container">About</a></li>
+          <li class="mob-nav-item"><a href="#contact-me">Contact</a></li>
```

Comment:
- [ ] The mobile menu is working fine, but on the desktop view, it is possible to see the close menu button. Kindly make it not be displayed on the desktop view and only appear when the user clicks on the menu hamburger button so the user can have a better experience.
 - [ ] The nav items are not well placed and can overlap the logo depending on the desktop width size. Please make sure that they are correctly positioned so your page can be more responsive and have a more professional look.
 -Your design:
<img width="899" alt="image" src="https://user-images.githubusercontent.com/61919617/194898381-e9ba187f-0621-49fd-9c59-62f81d99c3e6.png">